parsl/monitoring/db_logger.py

```diff
@@ -30,23 +32,90 @@ def emit(self, record):
         pass
 
 
-def get_db_logger(host='search-parsl-logging-test-2yjkk2wuoxukk2wdpiicl7mcrm.us-east-1.es.amazonaws.com',
-                  port=443,
-                  enable_es_logging=False,
-                  index_name="my_python_index",
-                  version='1.0.0',
+class LoggerConfig():
```

Comment:
I would prefer to call this a `MonitoringConfig` to make it clearer to users who associate logging with "something that writes text messages out to `parsl.log`" that this is specifically dealing with monitoring. Additionally, we might want to stick viz options in the same configuration, in which case LoggingConfig would make less sense. For similar reasons I vote `logger_type` -> `database_type` since both options (local or elasticsearch) are types of databases.